examples/rococo/assetHub/foreignAssetTransfers/reserve/assetHubMUSEToRhala.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * When importing from @substrate/asset-transfer-api it would look like the following
+ *
+ * import { AssetTransferApi, constructApiPromise } from '@substrate/asset-transfer-api'
+ */
+import { AssetTransferApi, constructApiPromise } from '../../../../../src';
+import { TxResult } from '../../../../../src/types';
+import { GREEN, PURPLE, RESET } from '../../../../colors';
+
+/**
+ * In this example we are creating a `polkadotXcm` pallet `transferAssetsUsingTypeAndThen` call to send MUSE (foreign asset with location `{"parents":"1","interior":{"X1":{"Parachain":"3369"}}}`)
+ * from a Rococo Asset Hub (System Parachain) account
+ * to a Rhala Testnet (ParaChain) account, where the `xcmVersion` is set to 4, and there is no
+ * `weightLimit` option provided which declares that the tx will allow unlimited weight to be used for fees.
```

Comment:
yes, here we need to use the `transferAssetsUsingTypeAndThen` in order to "force" using Asset Hub as `LocalReserve` 👍